dockerfiles/README.md

```diff
@@ -0,0 +1,116 @@
+Dockerfiles for Skosmos.
+
+## Running with Docker
+
+The following commands will build and tag the image it with `skosmos:test`,
+and run the container. The container name is `skosmos-web`, but you can customize
+the name, port, and other flags as necessary.
+
+```bash
```

Comment:
This is a pretty minor detail, but IMHO this block syntax is a bit hard to read when looking at the raw Markdown file in e.g. a text editor. I suggest using the [indented code block syntax](https://www.markdownguide.org/basic-syntax/#code-blocks) instead (although that may lose the bash specific syntax highlighting).